generate/generate.go

```diff
@@ -679,12 +679,12 @@ func lastCap() capability.Cap {
 	return last
 }
 
-func checkCap(c string, hostSpecific bool) error {
+func ValidCap(c string, hostSpecific bool) error {
```

Comment:
I think `ValidCap` should live in `validate/` and `generate.go` should import the `validate` package.